### DISCUSSION TITLE
blog page as homepage for site.

### DISCUSSION THREAD

**Discussion by prasadrgavande** [alshedivat/al-folio#2305] [Q&A] [answered]
Hi there,
I want to set blog page as home page for site. What i need to change in order to do this?

**Comment by george-gca** [comment, marked as answer]
Just change the permalink of the blog page to `/` and the permalink of the about page to something else.

**Reply by prasadrgavande** [reply]
Where to change those values ?

**Reply by george-gca** [reply]
In the respective pages frontmatter. I recommend you read the [CUSTOMIZE](https://github.com/alshedivat/al-folio/blob/master/CUSTOMIZE.md) part of our docs.
